docker/setup_configuration/data.yaml

```diff
@@ -131,3 +131,21 @@ sites_config:
   items:
     - domain: example.com
       name: Open Notificaties
+
+notifications_config_enable: true
+notifications_config:
+  notifications_api_service_identifier: notificaties-api
+  notification_delivery_max_retries: 1
+  notification_delivery_retry_backoff: 2
+  notification_delivery_retry_backoff_max: 3
+
+notifications_subscriptions_config_enable: true
+notifications_subscriptions_config:
+  items:
+    - identifier: autorisaties-subscription
```

Comment:
I checked it locally, it is indeed necessary to add the `Abonnement` via `notifications_abonnementen_config` as well. Complicating factor is this issue though: https://github.com/open-zaak/open-zaak/issues/1694

We can set up a subscription, but it won't work for more than an hour, because the JWT will expire :/